filebeat/harvester/forwarder.go

```diff
@@ -0,0 +1,57 @@
+package harvester
+
+import (
+	"errors"
+
+	"github.com/elastic/beats/filebeat/util"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/libbeat/processors"
+)
+
+type Outlet interface {
+	SetSignal(signal <-chan struct{})
+	OnEventSignal(data *util.Data) bool
+	OnEvent(data *util.Data) bool
+}
+
+type Forwarder struct {
+	Config     ForwarderConfig
+	Outlet     Outlet
+	Processors *processors.Processors
+}
+
+type ForwarderConfig struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ForwarderConfig should have comment or be unexported